app/controllers/services_controller.rb

```diff
@@ -3,21 +3,18 @@ class ServicesController < ApplicationController
   include ComponentConcern
 
   def index
-    component = component_by_klass_name(params)
-    @services = component.services
+    @services = Service.all
   end
 
   def new
-    @component = component_by_klass_name(params)
     @service_event = NewServiceEvent.new
   end
 
   def create
-    @component = component_by_klass_name(params)
     @service_event = NewServiceEvent.create(service_params)
 
-    if @service_event.valid? && @service_event.service.valid?
-      redirect_to polymorphic_url(@component)
+    if @service_event.valid?
```

Comment:
It would be nice to add a flash message to inform of what happened 
```
flash[:success] = t('common.action_successful', name:  @service_event.name, action: :created)
redirect_to admin_path(anchor: :services)
```